build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
     implementation('org.parboiled:parboiled_2.13:2.4.1')
     implementation('org.scalactic:scalactic_2.13:3.2.15')
     implementation("org.scala-lang.modules:scala-parallel-collections_2.13:0.2.0")
+    implementation('com.github.rjeschke:txtmark:0.13')
```

Comment:
We are generally very, very, very careful about including new libraries. In this case I guess it is almost required, but can I ask you to explore:

- How much does this increase the Flix har filesize by (it does not seem much).
- Can this library be trusted and is it the best/most popular/most tested library out there?
- What would we lose if we drop this library? (I guess some formatting?)

Reply:
- How much does this increase the Flix har filesize by (it does not seem much).
  - 62,259 bytes
- Can this library be trusted and is it the best/most popular/most tested library out there?
  - I believe it can be trusted, even though it hasn't been updated since 2015. It's apparently known to be pretty fast and minimal, and I get the impression that it is generally accepted. If we want something that's a bit more actively maintained I could look into Atlassian's [commonmark-java](https://github.com/commonmark/commonmark-java). This might be worth doing at some point anyway, to resolve the first point of this: https://github.com/flix/flix/pull/6316#issuecomment-1677031100.
- What would we lose if we drop this library? (I guess some formatting?)
  - We would lose the markdown support in the documentation. If we were willing to spend time on it, we could look into deciding on a minimal set of features we would like to support (like code and unordered lists) and implement that ourselves, but to me that sounds like a lot of effort to halfway replicate what we could get with a library.